internal/daemonset/daemonset.go

```diff
@@ -340,42 +364,73 @@ func OverrideLabels(labels, overrides map[string]string) map[string]string {
 }
 
 func MakeLoadCommand(spec kmmv1beta1.ModprobeSpec) []string {
-	loadCommand := []string{"modprobe"}
+	loadCommandShell := []string{
+		"/bin/sh",
+		"-c",
+	}
+
+	loadCommand := "modprobe"
 
 	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
-		return append(loadCommand, ra.Load...)
+		loadCommand = fmt.Sprintf("%s %s", loadCommand, strings.Join(ra.Load[:], " "))
+		return append(loadCommandShell, loadCommand)
+	}
+
+	if fw := spec.FirmwarePath; fw != "" {
+		loadCommand = fmt.Sprintf("cp -r %s %s && %s", fw, nodeVarLibFirmwarePath, loadCommand)
```

Comment:
How about doing this in an InitContainer instead?

Reply:
It wouldn't be symmetric with `preStop` hook, as there's not "`exitContainer`". The pod spec remains fairly simple with a single container that does everything.

Reply:
The `exitContainer` could be a regular `container` sleeping with a `preStop` hook. We would not have to move from an `exec`-type command to a shell one.
@yevgeny-shnaidman WDYT?

Reply:
A sleeping container could generate a race condition with the main container, as you wouldn't know which preStop gets executed first. If the module needs the firmware during is unload phase, it may not find it if the `exitContainer`'s preStop runs first.